File: Service/GroupMembershipManagement/Hosts/WebApi/Services.Messages/Responses/GetOrgLeaderDetailsResponse.cs
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.

using Services.Messages.Contracts.Responses;

namespace Services.Messages.Responses
{
    public class GetOrgLeaderDetailsResponse : ResponseBase
    {
        public int EmployeeId { get; set; }
        public int MaxDepth { get; set; }
    }
}

File: UI/web-app/src/models/GetOrgLeaderDetailsResponse.ts
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.

export interface GetOrgLeaderDetailsResponse {
    maxDepth: number;
    employeeId: number;
    objectId: string;
    partId: string;
    text: string;
}

export type OrgLeaderPickerPersona = {
    key: number;
    text: string;
    id: string;
};